ROOT fails to build on alma9 aarch64 with gcc13/14/15

### Check duplicate issues.

- [x] Checked for duplicates

### Description

Since yesterday, October 8, we get this error at the end of our builds of the ROOT master branch
```
make[6]: *** [CMakeFiles/modules_idx.dir/build.make:178: lib/modules.idx] Aborted (core dumped)
make[6]: *** Deleting file 'lib/modules.idx'
make[6]: Target 'CMakeFiles/modules_idx.dir/build' not remade because of errors.
make[5]: *** [CMakeFiles/Makefile2:10124: CMakeFiles/modules_idx.dir/all] Error 2
make[5]: Target 'all' not remade because of errors.
make[4]: *** [Makefile:156: all] Error 2
make[4]: Target 'default_target' not remade because of errors.
CMake Error at /build/jenkins/workspace/lcg_nightly_pipeline/build/projects/ROOT-HEAD/src/ROOT-HEAD-stamp/ROOT-HEAD-build-impl.cmake:20 (message):
```

on aarch64 alma9 gcc13. Full log here https://lcgapp-services.cern.ch/cdash/build/118442/files

Also fails for 
gcc14 https://lcgapp-services.cern.ch/cdash/build/118632/files
gcc15 https://lcgapp-services.cern.ch/cdash/build/118630/files

We don't test anything else, 6-36-patches builds fine.




### Reproducer

```
docker run -u root -it -e SHELL -e WORKSPACE=/workspace  --cpus=10 --security-opt seccomp:unconfined -v /cvmfs:/cvmfs -v /build:/build -v /ec/conf:/ec/conf   --network host  gitlab-registry.cern.ch/sft/docker/alma9:aarch64 bash

source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/Python-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/jsonmcpp/3.11.3/aarch64-el9-gcc15-opt/jsonmcpp-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/tiff/4.7.0/aarch64-el9-gcc15-opt/tiff-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/GSL/2.8/aarch64-el9-gcc15-opt/GSL-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/gl2ps/1.4.2/aarch64-el9-gcc15-opt/gl2ps-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/fftw3/3.3.10/aarch64-el9-gcc15-opt/fftw-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/cfitsio/4.6.2/aarch64-el9-gcc15-opt/cfitsio-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/xrootd/5.8.4/aarch64-el9-gcc15-opt/xrootd-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/Davix/0.8.10/aarch64-el9-gcc15-opt/Davix-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/tbb/2022.2.0/aarch64-el9-gcc15-opt/tbb-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/vdt/0.4.4/aarch64-el9-gcc15-opt/vdt-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/Vc/1.4.5/aarch64-el9-gcc15-opt/Vc-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/protobuf/5.28.3/aarch64-el9-gcc15-opt/protobuf-env.sh
source /cvmfs/sft.cern.ch/lcg/nightlies/dev4/Mon/ninja/1.11.1/aarch64-el9-gcc15-opt/ninja-env.sh

cd /build
mkdir testROOT
cd testROOT
git clone https://github.com/root-project/root.git
mkdir build install
cd build
cmake \
   -DCMAKE_BUILD_TYPE=Release \
   -DCMAKE_INSTALL_PREFIX=${PWD}/../install \
   -DCMAKE_CXX_STANDARD=20 \
   -Dcheck_connection=OFF \
   -DCINTLONGLINE=4096 \
   -DCINTMAXSTRUCT=36000 \
   -DCINTMAXTYPEDEF=36000 \
   -DPYTHON_EXECUTABLE=`which python` \
   -Dbuiltin_ftgl=ON \
   -Dbuiltin_glew=ON \
   -Dbuiltin_lz4=ON \
   -Dbuiltin_pcre=ON \
   -Dbuiltin_unuran=ON \
   -Dbuiltin_xxhash=ON \
   -Dbuiltin_zstd=ON \
   -Dcintex=ON \
   -Dexplicitlink=ON \
   -Dfail-on-missing=ON \
   -Dfftw3=ON \
   -Dfitsio=ON \
   -Dfortran=ON \
   -Dgdml=ON \
   -Dgenvector=ON \
   -Dhttp=ON \
   -Dmathmore=ON \
   -Dopengl=ON \
   -Dpyroot=ON \
   -Dpythia8=OFF \
   -Dr=OFF \
   -Dreflex=ON \
   -Droofit=ON \
   -Droofit_multiprocess=OFF \
   -Droot7=ON \
   -Dssl=ON \
   -Dtesting=OFF \
   -Dunfold=ON \
   -Dunuran=ON \
   -Dxft=ON \
   -Dxml=ON \
   -Dxrootd=ON \
   -Dzlib=ON \
   -Dvc=ON \
   -Dtmva-sofie=ON \
   -Dmysql=OFF \
   -Ddavix=ON \
   -Dbuiltin_gif=ON \
   -Dbuiltin_jpeg=OFF \
   -Dbuiltin_png=OFF \
   -Droottest=OFF \
   -Dbuiltin_nlohmannjson=OFF \
   -G Ninja ../root
ninja

```

In the nightly builds we use lcgcmake to build everything, and we do not use the view, but this way should be easier to change files and rebuild and see if things are working.


ninja ends with
```
[2/3] Generating lib/modules.idx
FAILED: lib/modules.idx /build/sailer/testRoot/build2/lib/modules.idx 
cd /build/sailer/testRoot/build2/lib && /usr/bin/cmake -E remove -f modules.idx modules.timestamp && LD_LIBRARY_PATH=/build/sailer/testRoot/build2/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/absl/20230802.1/aarch64-el9-gcc15-opt/lib64/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/absl/20230802.1/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/setuptools/75.8.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/pip/23.3.1/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/protobufpy/5.28.3/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/protobuf/5.28.3/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/protobuf/5.28.3/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/vdt/0.4.4/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Vc/1.4.5/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Vc/1.4.5/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/tbb/2022.2.0/aarch64-el9-gcc15-opt/lib64/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/tbb/2022.2.0/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libxml2/2.10.4/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libxml2/2.10.4/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/blas/0.3.29.openblas/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/blas/0.3.29.openblas/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/cython/3.0.12/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/meson/1.3.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/setuptools/75.8.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/pip/23.3.1/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/numpy/2.1.3/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Boost/1.89.0/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Boost/1.89.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Davix/0.8.10/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/blas/0.3.29.openblas/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/blas/0.3.29.openblas/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/cython/3.0.12/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/meson/1.3.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/numpy/2.1.3/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Boost/1.89.0/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Boost/1.89.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Davix/0.8.10/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libxml2/2.10.4/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libxml2/2.10.4/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/setuptools/75.8.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/pip/23.3.1/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/xrootd/5.8.4/aarch64-el9-gcc15-opt/lib64/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/xrootd/5.8.4/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/xrootd/5.8.4/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/cfitsio/4.6.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/fftw3/3.3.10/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/fftw3/3.3.10/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/png/1.6.50/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/gl2ps/1.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/GSL/2.8/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/tiff/4.7.0/aarch64-el9-gcc15-opt/lib64/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/tiff/4.7.0/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib ROOTIGNOREPREFIX=1 ROOT_HIST=0 /build/sailer/testRoot/build2/bin/root.exe -l -q -b
ninja: build stopped: subcommand failed.
```

And if you run this by hand you get
```
cd /build/sailer/testRoot/build2/lib && /usr/bin/cmake -E remove -f modules.idx modules.timestamp && LD_LIBRARY_PATH=/build/sailer/testRoot/build2/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/absl/20230802.1/aarch64-el9-gcc15-opt/lib64/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/absl/20230802.1/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/setuptools/75.8.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/pip/23.3.1/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/protobufpy/5.28.3/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/protobuf/5.28.3/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/protobuf/5.28.3/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/vdt/0.4.4/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Vc/1.4.5/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Vc/1.4.5/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/tbb/2022.2.0/aarch64-el9-gcc15-opt/lib64/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/tbb/2022.2.0/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libxml2/2.10.4/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libxml2/2.10.4/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/blas/0.3.29.openblas/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/blas/0.3.29.openblas/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/cython/3.0.12/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/meson/1.3.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/setuptools/75.8.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/pip/23.3.1/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/numpy/2.1.3/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Boost/1.89.0/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Boost/1.89.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Davix/0.8.10/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/blas/0.3.29.openblas/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/blas/0.3.29.openblas/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/cython/3.0.12/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/meson/1.3.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/numpy/2.1.3/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Boost/1.89.0/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Boost/1.89.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Davix/0.8.10/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libxml2/2.10.4/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libxml2/2.10.4/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/setuptools/75.8.0/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/pip/23.3.1/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/xrootd/5.8.4/aarch64-el9-gcc15-opt/lib64/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/xrootd/5.8.4/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/xrootd/5.8.4/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/cfitsio/4.6.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/fftw3/3.3.10/aarch64-el9-gcc15-opt/lib/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/fftw3/3.3.10/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/png/1.6.50/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/gl2ps/1.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/GSL/2.8/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/tiff/4.7.0/aarch64-el9-gcc15-opt/lib64/cmake:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/tiff/4.7.0/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib64:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/libffi/3.4.2/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/sqlite/3320300/aarch64-el9-gcc15-opt/lib:/cvmfs/sft-nightlies.cern.ch/lcg/nightlies/dev4/Mon/Python/3.12.11/aarch64-el9-gcc15-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/15.2.0-35657/aarch64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/aarch64-el9/lib ROOTIGNOREPREFIX=1 ROOT_HIST=0 /build/sailer/testRoot/build2/bin/root.exe -l -q -b
Aborted (core dumped)
```

And in the core dump I get
```
(gdb) bt
#0  0x0000ffff926d3db8 in __pthread_kill_implementation () from /lib64/libc.so.6
#1  0x0000ffff9268c6bc in raise () from /lib64/libc.so.6
#2  0x0000ffff92678fb4 in abort () from /lib64/libc.so.6
#3  0x0000ffff932757bc in __deregister_frame_info_bases (begin=<optimized out>) at /workspace/build/contrib/gcc-15.2.0/src/gcc/15.2.0/libgcc/unwind-dw2-fde.c:296
#4  __deregister_frame_info_bases (begin=<optimized out>) at /workspace/build/contrib/gcc-15.2.0/src/gcc/15.2.0/libgcc/unwind-dw2-fde.c:230
#5  0x0000ffff93275838 in __deregister_frame (begin=<optimized out>) at /workspace/build/contrib/gcc-15.2.0/src/gcc/15.2.0/libgcc/unwind-dw2-fde.c:311
#6  0x0000ffff8e010458 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#7  0x0000ffff8de6b79c in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#8  0x0000ffff8de55b38 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#9  0x0000ffff8de72e20 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#10 0x0000ffff8de38fa4 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#11 0x0000ffff8de45fe0 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#12 0x0000ffff8de262c4 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#13 0x0000ffff8de26640 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#14 0x0000ffff8c5871d8 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#15 0x0000ffff8c57edd4 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#16 0x0000ffff8c4ffcd8 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#17 0x0000ffff8c4fffb0 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#18 0x0000ffff8c3f403c in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#19 0x0000ffff8c3f4250 in ?? () from /build/sailer/testRoot/build2/lib/libCling.so
#20 0x0000ffff92e5ae00 in TROOT::~TROOT() () from /build/sailer/testRoot/build2/lib/libCore.so
#21 0x0000ffff9268ef5c in __run_exit_handlers () from /lib64/libc.so.6
#22 0x0000ffff9268f030 in exit () from /lib64/libc.so.6
#23 0x0000ffff92f76c30 in TUnixSystem::Exit(int, bool) () from /build/sailer/testRoot/build2/lib/libCore.so
#24 0x0000ffff92e04524 in TApplication::Terminate(int) () from /build/sailer/testRoot/build2/lib/libCore.so
#25 0x0000ffff93341e30 in TRint::Run(bool) () from /build/sailer/testRoot/build2/lib/libRint.so
#26 0x0000000000400e34 in main ()
```

### ROOT version

master branch from today (October 8)

### Installation method

building from source

### Operating system

Alma Linux 9, aarch64

### Additional context

_No response_

--- a/interpreter/cling/lib/Interpreter/IncrementalJIT.cpp
+++ b/interpreter/cling/lib/Interpreter/IncrementalJIT.cpp
@@ -19,8 +19,6 @@
 #include <clang/Basic/TargetOptions.h>
 #include <clang/Frontend/CompilerInstance.h>
 
-#include <llvm/ExecutionEngine/JITLink/EHFrameSupport.h>
-#include <llvm/ExecutionEngine/Orc/EHFrameRegistrationPlugin.h>
 #include <llvm/ExecutionEngine/Orc/JITTargetMachineBuilder.h>
 #include <llvm/ExecutionEngine/Orc/ObjectLinkingLayer.h>
 #include <llvm/ExecutionEngine/Orc/RTDyldObjectLinkingLayer.h>
@@ -505,8 +503,6 @@ IncrementalJIT::IncrementalJIT(
       unsigned PageSize = cantFail(sys::Process::getPageSize());
       auto ObjLinkingLayer = std::make_unique<ObjectLinkingLayer>(
           ES, std::make_unique<ClingJITLinkMemoryManager>(PageSize));
-      ObjLinkingLayer->addPlugin(std::make_unique<EHFrameRegistrationPlugin>(
-          ES, std::make_unique<InProcessEHFrameRegistrar>()));
       return ObjLinkingLayer;
     }
 

